workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/SendPlayerHeartbeatRequestSystem.cs

```diff
@@ -20,19 +21,28 @@ namespace Improbable.Gdk.PlayerLifecycle
     [UpdateBefore(typeof(SpatialOSUpdateGroup))]
     public class SendPlayerHeartbeatRequestSystem : ComponentSystem
     {
-        private struct Data
+        private float timeOfNextHeartbeat = Time.time + PlayerLifecycleConfig.PlayerHeartbeatIntervalSeconds;
```

Comment:
Don't use Time.time it loses accuracy quite quickly. But it doesn't look like we are using this anywhere.

Reply:
Ah found it's use. Counting down might be slightly better until the timer module gets more love.